xapian-letor/tests/api_letor.cc

```diff
@@ -833,3 +914,456 @@ DEFINE_TESTCASE(ndcg_score_test, generated)
     unlink("ndcg_score_test.txt");
     return true;
 }
+
+// Test createfeaturevector method for TfFeature
+DEFINE_TESTCASE(createfeaturevector_tffeature, generated)
+{
+    vector<Xapian::Feature*> f;
+    Xapian::TfFeature* f1 = new Xapian::TfFeature();
+    f.push_back(f1);
+
+    Xapian::FeatureList fl(f);
+    Xapian::Database db = get_database("db_index_three_documents",
+				       db_index_three_documents);
+    Xapian::Enquire enquire(db);
+    enquire.set_query(Xapian::Query("score"));
+    Xapian::MSet mset;
+    mset = enquire.get_mset(0, 10);
+
+    TEST(!mset.empty());
+
+    Xapian::QueryParser queryparser;
+    queryparser.set_stemmer(Xapian::Stem("en"));
+    queryparser.set_stemming_strategy(queryparser.STEM_ALL_Z);
+    queryparser.add_prefix("title", "S");
+    queryparser.add_prefix("description", "XD");
+
+    string querystring = "title:score description:score score";
+    Xapian::Query query = queryparser.parse_query(querystring);
```

Comment:
This needs a comment explaining why we're doing this rather than just using `Query("score")`.

Reply:
fixed in a fixup commit.